trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Lift/VLM.py

```diff
@@ -500,6 +512,7 @@ def VLM(conditions,settings,geometry):
     # STEP 13: Pack outputs
     # ------------------ --------------------------------------------------------------------     
     precision      = settings.floating_point_precision
+    VD.gamma       = GAMMA
```

Comment:
No. This is very serious. Two reasons:

GAMMA is huge we can't save it arbitrarily. This could be well over 10's of GB's for a complex configuration. Many computers don't have that kind of storage/memory space for something users won't need 99% of the time.

GAMMA should only be accessible after running the analysis and saving the results. Saving this to VD, which is the Vortex Distribution makes absolutely no sense. GAMMA is a condition and VD is geometric. Without reference conditions (AOA/Mach) is not meaningful. Every segment this would be overwritten anyways.